pywbemtools/pywbemcli/_cmd_statistics.py

```diff
@@ -129,20 +128,21 @@ def statistics_server_on(context):
 @click.pass_obj
 def statistics_server_off(context):
     """
-    Disable statistics on current server.
+    Disable server maintained statistics.
 
-    This command turns off the gathering of statistics in the WBEM
-    server and the return of server response times for inclusion in
-    pywbemcli maintained statistics. The gathering of server statistics and the
-    returning of server response times may not be implemented by all WBEM
-    servers.
+    This command deactivates the gathering of statistics in the current
+    WBEM server and also deactivates the returning of server response times in
```

Comment:
probably the also not needed.  the and covers it

Reply:
We describe the command to activate and deactivate the server statistics. From that perspective, the enabling of the returning of server response time (which becomes part of the client statistics) is not what one would expect and is more like a side effect. That's why I used 'also'. NOT CHANGED.

Reply:
We agreed in the call today this is ok to go.